src/Configuration.php

```diff
@@ -11,7 +10,6 @@
  */
 class Configuration
 {
-
```

Comment:
GH Actions are complaining on this file, @hassansin. And it might be real, the defined field is called `$retry`, not `$retries` :thinking:

Reply:
@pkopac these are issues from Static Analysis tool, there's quite a lot of them. I'll address them in a separate PR.